src/ScriptEditor/ui/ScriptEditorRoot.tsx

```diff
@@ -892,8 +892,8 @@ export function Root(props: IProps): React.ReactElement {
             {ram}
           </Button>
           <Button onClick={save}>Save (Ctrl/Cmd + s)</Button>
-          <Button onClick={props.router.toTerminal}>Close (Ctrl/Cmd + b)</Button>
-          <Typography sx={{ mx: 1 }}>
+          <Button sx={{ mx: 1 }} onClick={props.router.toTerminal}>Terminal (Ctrl/Cmd + b)</Button>
```

Comment:
I think this is "truer" but is it better? Maybe something like `Go to terminal` would work? Or would that be to verbose?

Reply:
i wanted to the same originally but theres not enough real estate, if we make it a multiline button flex then resizes the other buttons too and they look gross. happy to try something else if you have any ideas